cns/restserver/restserver.go

```diff
@@ -1090,6 +1090,14 @@ func (service *HTTPRestService) getNetworkContainerByOrchestratorContext(w http.
 		return
 	}
 
+	// getNetworkContainerByOrchestratorContext gets called for multitenancy and
+	// setting the SDNRemoteArpMacAddress regKey is essential for the multitenancy
+	// to work correctly in case of windows platform. Return if there is an error
+	if err = platform.SetSdnRemoteArpMacAddress(); err != nil {
```

Comment:
Is there another location we may set this? I think getNetworkContainerByOrchestratorContext gets called fairly often.
We're only checking whether it's already set so uty if whether you want to move it or not...

Reply:
Unfortunately there is no other place to do this. We don't know if it's going to be multitenancy when we get SetOrchestratorContext. Even if we call it here, it won't be a performance issue because it will return immediately if the flag - sdnRemoteArpMacAddressSet is set to true.